fundingmanager.go

```diff
@@ -2785,7 +2796,17 @@ func (f *fundingManager) handleInitFundingMsg(msg *initFundingMsg) {
 	// wallet doesn't have enough funds to commit to this channel, then the
 	// request will fail, and be aborted.
 	//
-	// TODO(roasbeef): signal tweakless or not
+	// Before we init the channel, we'll also check to see if we've
+	// negotiated the new tweakless commitment format. This is only the
```

Comment:
So there is really no negotiation here? Assumption is that as long as both supports it, it will be used.

Reply:
Yeah we do a logical AND w.r.t the existence of the feature bit in the init message. I had suggested in the past that we also let them signal it by using a new funding flag, but we have a single unified way by just using the feature bits. Put simply: only if both nodes support it is it used. Arguably, being able to set that value on the fly would make it easier for our integration tests, as we could use the same nodes (no additional flag) and just use the RPC system to modulate if we're doing tweakless or not.